src/domBackedDomFacade.js

```diff
@@ -2,33 +2,39 @@
  * @constructor
  * @implements {IDomFacade}
  */
-function ReadOnlyDomFacade () {
+function DomBackedDomFacade () {
+	/**
+	 * Defines the ordering of detached nodes, to ensure stable sorting of unrelated nodes.
+	 *
+	 * @type {!Array<!./selectors/dataTypes/Value>}
+	 */
+	this.orderOfDetachedNodes = [];
```

Comment:
This may leak memory, unless we set its length to 0 after each evaluateXPath, or wrap DomBackedDomFacade in a temporary DomFacade.

Reply:
The domFacade is now always wrapped.

Reply:
Does that mean that this field is now unused (and can be removed)?